src/workflows-app/components/TroubleshootingBox.js

```diff
@@ -0,0 +1,50 @@
+import { div, h, span } from 'react-hyperscript-helpers';
+import { ClipboardButton } from 'src/components/ClipboardButton';
+import { Link } from 'src/components/common';
+import { icon } from 'src/components/icons';
+import colors from 'src/libs/colors';
+import * as Nav from 'src/libs/nav';
+
+export const TroubleshootingBox = ({ name, namespace, logUri, submissionId, workflowId, showLogModal }) => {
```

Comment:
Tiny request would be to center + evenly space the `Execution Log` and `Execution Directory` links in the `TroubleshootingBox`. I personally think it would look better than having them left justified.